tests/test-backend-ops.cpp

```diff
@@ -607,28 +945,35 @@ struct test_case {
         GGML_ASSERT(ctx);
 
         ggml_tensor * out = build_graph(ctx.get());
-
-        if (op_name != nullptr && op_desc(out) != op_name) {
+        std::string current_op_name = op_desc(out);
+        if (op_name != nullptr && current_op_name != op_name) {
             //printf("  %s: skipping\n", op_desc(out).c_str());
             return true;
         }
 
-        int len = printf("  %s(%s): ", op_desc(out).c_str(), vars().c_str());
-        fflush(stdout);
-
         // check if backends support op
         if (!ggml_backend_supports_op(backend, out)) {
-            printf("not supported\n");
-            return true;
-        }
+            // Create test result for unsupported performance test
+            test_result result;
+            result.backend_name = ggml_backend_name(backend);
+            result.op_name = current_op_name;
+            result.op_params = vars();
+            result.test_mode = "perf";
+            result.supported = false;
+            result.passed = false;
+            result.error_message = "not supported";
+            result.time_us = 0.0;
+            result.flops_per_sec = 0.0;
+            result.bandwidth_gb_s = 0.0;
+            result.memory_kb = 0;
+            result.n_runs = 0;
```

Comment:
I think it would be better set properties via a constructor instead of setting them after object creation.